model_zoo/models/molecular_autoencoder_candle_pilot2/model_conv_molecular_autoencoder_pilot2.prototext

```diff
@@ -165,46 +153,31 @@ model {
       epsilon: 1e-5
     }
   }
-
-  #######
-  # RELU relu10
-  #######
-  layer {
-    name: "d_relu_fc1"
-    data_layout: "data_parallel"
-    relu {
-    }
-  }
-
   layer {
-    name: "d_reshape1"
+    parents: "decode2_bn"
+    name: "decode2"
     data_layout: "data_parallel"
-    reshape {
-      num_dims: 2
-      dims: "1024 11"
```

Comment:
What was the reason for removing the reshape layer?

Reply:
We set "decode2_fc" to have the same output dimensions as "encode1", which is already a 2D tensor.

Reply:
Okay.